src/Umbraco.Web.UI.Client/src/packages/rte/components/rte-base.element.ts

```diff
@@ -21,6 +21,15 @@ import type {
 	UmbPropertyEditorConfigCollection,
 } from '@umbraco-cms/backoffice/property-editor';
 
+/**
+ * The abstract base class that is used as a base for the rich-text-editor component.
+ * @cssprop --umb-rte-width - The width of the rich-text-editor (default: unset)
+ * @cssprop --umb-rte-min-width - The minimum width of the rich-text-editor (default: unset)
+ * @cssprop --umb-rte-max-width - The maximum width of the rich-text-editor (default: 100%)
+ * @cssprop --umb-rte-height - The height of the rich-text-editor (default: 100%)
+ * @cssprop --umb-rte-min-height - The minimum height of the rich-text-editor (default: 100%)
+ * @cssprop --umb-rte-max-height - The maximum height of the rich-text-editor (default: 100%)
+ */
```

Comment:
Not sure this update should be in this PR - or how it got there!  But to keep things clean would be better to just update files that are necessary to meet the issue.